src/apis/cart.ts

```diff
@@ -0,0 +1,46 @@
+import { CartProduct, CartItemsResponse } from '../types/cart';
+import { apiClient } from './apiClient';
+
+export const getCartItems = async (): Promise<CartItemsResponse> => {
+  try {
+    return await apiClient<CartItemsResponse>('/cart-items?page=0&size=20');
```

Comment:
응답 타입은 자연스럽게 추론된답니다!

```suggestion
export const getCartItems = async () => {
  try {
    return await apiClient<CartItemsResponse>('/cart-items?page=0&size=20');
```

Reply:
확인했어요! 응답 타입 제거했습니다~!

[refactor: 불필요 타입 제거](https://github.com/woowacourse/react-shopping-cart/pull/405/commits/fa12572a998c3be982883649a0c952d8a9f4538c)